demos/ripple.html

```diff
@@ -14,85 +14,104 @@
   limitations under the License
 -->
-<html>
+<html class="mdl-typography">
   <head>
     <meta charset="utf-8">
     <title>MDL Ripple Demo</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+    <script>
+      if (!window.CSS || !window.CSS.supports || !window.CSS.supports('--css-vars', 'yes')) {
+        document.documentElement.classList.add('unsupported');
+      }
+    </script>
     <style type="text/css">
-    .surface {
-      box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
-      0px 1px 1px 0px rgba(0, 0, 0, 0.14),
-      0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+      .demo-surface {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 200px;
+        height: 100px;
+        padding: 1rem;
+        cursor: pointer;
+        user-select: none;
+        -webkit-user-select: none;
+      }
+
+      .demo-surface[data-mdl-ripple-is-unbounded] {
+        width: 40px;
+        height: 40px;
+        padding: 0;
+        border-radius: 50%;
+      }
+
+      button.demo-surface {
+        background: none;
+        border: none;
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
+      }
+
+      section {
+        margin: 48px;
+      }
+
+      body {
+        font-family: sans-serif;
+      }
 
-      position: relative;
-    }
-     section {
-       margin: 48px;
-     }
-     body {
-       font-family: sans-serif;
-     }
-     button {
-       margin: 16px 0;
-     }
+      #unsupported-msg {
+        padding: 1rem;
+        border: 1px solid red;
+        color: red;
+        border-radius: 4px;
+        display: none;
+      }
+
+      .unsupported #unsupported-msg {
+        display: block;
+      }
+
+      .unsupported main {
+        opacity: .2;
+        pointer-events: none;
+        user-select: none;
+        -webkit-user-select: none;
+      }
     </style>
   </head>
   <body>
+    <aside id="unsupported-msg">
+      It appears your browser does not support custom properties, or has a broken implementation.
+      Please try this in Desktop Chrome or Desktop Firefox.
+    </aside>
     <main>
+      <h1>MDL Ripple</h1>
       <section>
         <h2>Bounded</h2>
-        <div class="surface auto" style="overflow: hidden; height: 100px; width: 200px"></div>
-        <button>Trigger Ripple</button>
+        <div class="mdl-ripple-surface demo-surface mdl-elevation--z2" tabindex="0">
+          Interact with me!
+        </div>
       </section>
       <section>
         <h2>Unbounded</h2>
-        <div class="surface auto" style="height: 40px; width: 40px;" bounded="false" max-radius="50"></div>
-        <button>Trigger Ripple</button>
+        <div class="mdl-ripple-surface demo-surface material-icons" data-mdl-ripple-is-unbounded
```

Comment:
The data attribute approach seemed more straightforward for configuration to me. Rather than having to parse out configuration info from a list of classnames, it is explicitly specified by using a data attribute.  WDYT?